tests/test_request_timeout.py

```diff
@@ -5,9 +5,24 @@
 from sanic.response import text
 from sanic.config import Config
 import aiohttp
-from aiohttp import TCPConnector
+from aiohttp import TCPConnector, ClientResponse
 from sanic.testing import SanicTestClient, HOST, PORT
 
+try:
+    try:
+        import packaging # direct use
+    except ImportError:
+        # setuptools v39.0 and above.
+        try:
+            from setuptools.extern import packaging
+        except ImportError:
+            # Before setuptools v39.0
+            from pkg_resources.extern import packaging
+    version = packaging.version
+except ImportError:
+    raise RuntimeError("The 'packaging' library is missing.")
```

Comment:
does it make sense to have a packaging defined once for sanic hand other tests import it? or would importing non sanic modules from sanic modules be an anti pattern?